[Term Entry] Kotlin Math .acos() (#3613)
Codecademy/docs: content/kotlin/concepts/math-methods/terms/acos/acos.md

---
Title: '.acos()'

> Description: 'Calculates the arccosine or reverse cosine of a value, returning the angle in radians.'

Author: should i also include inverse cosine?

Description: 'Calculates the arccosine or reverse cosine or inverse cosine of a value, returning the angle in radians.'

Subjects:
  - 'Code Foundation'
  - 'Computer Science'
Tags:
  - 'Math'
  - 'Functions'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

The **`.acos()`** method in Kotlin's `Math` class is used to calculate the
arccosine of a given value, returning the corresponding angle in radians.

## Syntax

```pseudo

> val angle = Math.acos(value)

Reviewer: ```suggestion
import kotlin.math.*
val angle = acos(value)
```

```

> `value`: The value for which you want to calculate the arccosine. It should be
> in the range [-1, 1].

Reviewer: ```suggestion
`value`: The given value to be used in the arccosine calculation. The value should be in the range `-1` to `1`.
```

> The `.acos()` method returns a value of type `double` representing the angle in
> radians. The result is in the range [0, PI].

Reviewer: ```suggestion
The `.acos()` method returns a value of type `double` representing the angle in radians. The result will be in the range `0` to `PI`.
```

## Example

> This example shows how to use the `.acos()` method to calculate the arccosine of
> a value:

Reviewer: ```suggestion
This example shows how to use the `acos()` method to calculate the arccosine of a value:
```

```kotlin
import kotlin.math.*

fun main() {
    val value = 0.5

>     val angle = Math.acos(value)

Reviewer: ```suggestion
    val angle = acos(value)
```

    println("The arccosine of $value is $angle radians.")
}
```

The output of this code will be:

```shell

> 1.0471975511965979 rad

Reviewer: this output doesn't match the print statement
